docker-compose.e2e-tests.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.7'
+
+services:
+  e2e_playwright:
+    build:
+      context: .
+      dockerfile: packages/e2e-tests/Dockerfile
+      args:
+        gid: ${GID}
+        uid: ${UID}
+    environment:
+      - NODE_ENV=test
+      - NODE_TLS_REJECT_UNAUTHORIZED=0
+      - DC_ACCOUNTS_DIR=../../e2e-tests/data/accounts
+    volumes:
+      - ./:/app
```

Comment:
What is the point of having a bind mount? Not having to rebuild the image every time the code changes?

Reply:
Yes, the image should only be rebuilt if we change to another playwright version. And the current dir is mounted to /app/ in the container to have always the current sources